src/infrastructure/router/router.go

```diff
@@ -0,0 +1,25 @@
+package router
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/spf13/viper"
+	"golang-bootcamp-2020/controllers"
+)
+
+var (
+	router = gin.Default()
+)
+
+//Setting api router
+func StartRouter(c controllers.AppController) error {
```

Comment:
By convention, all the descriptions should start with the name of the struct, interface, function, etc name
Please change it